src/pages/BusinessBankAccount/NewPage.js

```diff
@@ -0,0 +1,183 @@
+import _ from 'underscore';
+import React from 'react';
+import {View, Text, TouchableOpacity} from 'react-native';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import MenuItem from '../../components/MenuItem';
+import {Bug, Lock} from '../../components/Icon/Expensicons';
+import styles from '../../styles/styles';
+import TextLink from '../../components/TextLink';
+import Button from '../../components/Button';
+import Icon from '../../components/Icon';
+import colors from '../../styles/colors';
+import Navigation from '../../libs/Navigation/Navigation';
+import Permissions from '../../libs/Permissions';
+import CONST from '../../CONST';
+import TextInputWithLabel from '../../components/TextInputWithLabel';
+import Checkbox from '../../components/Checkbox';
+import AddPlaidBankAccount from '../../components/AddPlaidBankAccount';
+
+class BusinessBankAccountNewPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.toggleTerms = this.toggleTerms.bind(this);
+        this.addManualAccount = this.addManualAccount.bind(this);
+
+        this.state = {
+            addMethodSelection: undefined,
+            hasAcceptedTerms: false,
+            routingNumber: undefined,
+            accountNumber: undefined,
+        };
+    }
+
+    toggleTerms() {
+        this.setState(prevState => ({
+            hasAcceptedTerms: !prevState.hasAcceptedTerms,
+        }));
+    }
+
+    canSubmitManually() {
+        // @TODO add better validation
```

Comment:
I'm thinking at least check for a 9 digit routing number + that both are all numbers? I'll look at how web-secure is handling this.